tests/test_kendall_tau_distance.py

```diff
@@ -33,6 +47,25 @@ def test_pairwise_distance_matrix():
     assert D[1][2] < D[1][0]
 
 
+def test_dask_pairwise_distance_matrix():
+    client = Client()
+    r1 = [0.05, 0.2, 0.7, 0.05]
+    r2 = [0.23, 0.24, 0.26, 0.27]
+    r3 = [0.22, 0.24, 0.26, 0.28]
+    rankings = np.array([r1, r2, r3])
+
+    D = pairwise_distance_matrix(rankings)
+    # check symmetry, within floating point rounding margin
+    assert (D[0][1] - D[1][0]) < 1e-9
+    # check diagonal is zero
+    assert D[1][1] == 0
+    assert D[2][2] == 0
+    # distance between r2 and r3 is closer than r2 and r1
+    assert D[1][2] < D[1][0]
+    client.close()
```

Comment:
This is testing the same thing as the function above but exclusively on dask but the above tests both dask and numpy. I say remove the dask pieces from the above function and just ahve dask tested in this function exclusively.